README.md

```diff
@@ -98,4 +105,79 @@ automatically remove set all auto-merge labels from the PR
 _Note_: at the moment the Dlang-Bot doesn't store _any_ authentication token, so
 it can't perform actions on behalf of others.
 
-<a name="canceling-state" />
+<a name="warn-common-mistakes" />
```

Comment:
FWIW there are Markdown TOC generators such as https://github.com/jonschlinkert/markdown-toc which reuse GitHub's existing anchors.

I remember I used one for https://github.com/CyberShadow/aconfmgr#table-of-contents, though it needed some minor tweaks.